projects/meetme.md

```diff
@@ -0,0 +1,10 @@
+MeetMe
+--------------------
+
+### Background:
+
+Automating the process of scheduling and keeping track of meetings has been around for a long time, especially in professional environments. Typically, organizations make use of products such as Outlook or Google Calendar to achieve this. However, these either require all users to be on a certain app/network, or require users to share contacts/details with each other, which is troublesome, especially for one-off meetings. They are also not tailored for individuals.
+MeetMe is an app that syncs with all your calendars to seamlessly schedule meetings, whether a professional one or for drinks with friends.
```

Comment:
I am pretty sure that in Markdown this needs to have another empty line if you want it on a separate line.